### DISCUSSION TITLE
Override SEPA-Xml bundle

### DISCUSSION THREAD

**Discussion by aghar-saif-eddine** [symfony/symfony#44247] [Q&A]
I am using sepa-XML(https://github.com/php-sepa-xml/php-sepa-xml) to generate my XML file. I want to add some fields to my XML. So my question is can I override this bundle ? and if I can does any examples.
Thanks.

**Comment by stof** [comment]
The symfony discussions are not a support channel for unrelated packages. You should ask this to the php-sepa-xml project. We cannot provide support for the whole ecosystem (that we don't even know)
